crates/distribution-types/src/installed.rs

```diff
@@ -190,3 +190,12 @@ impl InstalledMetadata for InstalledDist {
         }
     }
 }
+
+impl InstalledDirectUrlDist {
+    /// Get the direct url json for a direct url dist
+    /// useful function for installers that want to check the
+    /// `direct_url.json` metadata
+    pub fn direct_url(&self) -> Result<Option<pypi_types::DirectUrl>, anyhow::Error> {
+        InstalledDist::direct_url(&self.path)
```

Comment:
Can you not use this method directly in Pixi?

Reply:
For sure thats what I did :). However, other files can be accessed like `installer`, `metadata` etc. This one is the only one thats not public, so I thought it might make sense to expose.

Reply:
Hey @charliermarsh, any thoughts on this? The `InstalledDist::direct_url` is not public either, not sure if that was clear. I've made A PR if you like that more: https://github.com/astral-sh/uv/pull/2584

Reply:
Okay thanks, I prefer that PR, sorry for the delay!